dw-menu.js

```diff
@@ -19,6 +23,20 @@ export class DwMenu extends DwCompositeDialog {
     css`
       :host {
         display: block;
+        --dw-dialog-content-padding: 0;
+      }
+
+      :host([type="popover"]) .dialog__content {
+        padding: var(--dw-menu-content-padding, 0);
+      }
+
+      dw-list-item:not([disabled]) {
+        --dw-icon-color: var(--mdc-theme-text-secondary-on-background, rgba(0, 0, 0, 0.6));
+      }
+
+      dw-list-item:not([disabled])[danger] {
```

Comment:
In case an item is rendered in danger colour, shouldn't it's ripple also be in danger? Is this considered or works that way?